pkg/accounting/tally/tally.go

```diff
@@ -56,122 +52,80 @@ func (t *tally) Run(ctx context.Context) (err error) {
 	defer mon.Task()(&ctx)(&err)
 
 	for {
-		err = t.identifyActiveNodes(ctx)
+		err = t.calculateStaticData(ctx)
+		if err != nil {
+			t.logger.Error("calculateStaticData failed", zap.Error(err))
+		}
+		err = t.Query(ctx)
 		if err != nil {
-			zap.L().Error("Tally failed", zap.Error(err))
+			t.logger.Error("Query for bandwith failed", zap.Error(err))
 		}
 
 		select {
 		case <-t.ticker.C: // wait for the next interval to happen
 		case <-ctx.Done(): // or the tally is canceled via context
-			_ = t.db.Close()
 			return ctx.Err()
 		}
 	}
 }
 
-// identifyActiveNodes iterates through pointerdb and identifies nodes that have storage on them
-func (t *tally) identifyActiveNodes(ctx context.Context) (err error) {
+// calculateStaticData iterates through the pieces on pointerdb and calculates 
+// the amount of at-rest data stored on each respective node
+func (t *tally) calculateStaticData(ctx context.Context) (err error) {
 	defer mon.Task()(&ctx)(&err)
-
-	rt, err := t.kademlia.GetRoutingTable(ctx)
-	if err != nil {
-		return Error.Wrap(err)
-	}
-	self := rt.Local()
-	identity := &provider.FullIdentity{} //do i need anything in here?
-	client, err := node.NewNodeClient(identity, self, t.kademlia)
-	if err != nil {
-		return Error.Wrap(err)
-	}
-
+	t.logger.Debug("entering pointerdb iterate within identifyActiveNodes")
+	var nodeData = make(map[storj.NodeID]int64)
 	err = t.pointerdb.Iterate(ctx, &pb.IterateRequest{Recurse: true},
 		func(it storage.Iterator) error {
 			var item storage.ListItem
-			lim := t.limit
-			if lim <= 0 || lim > storage.LookupLimit {
-				lim = storage.LookupLimit
-			}
-			for ; lim > 0 && it.Next(&item); lim-- {
+			for it.Next(&item) {
```

Comment:
Let's make a separate issue for proper batching in `pointerdb.Iterate`

Reply:
https://storjlabs.atlassian.net/browse/V3-996